mailserver.env

```diff
@@ -332,6 +332,17 @@ REPORT_SENDER=
 LOGROTATE_INTERVAL=weekly
 
 
+# Choose how Postfix handles DSN (Delivery Status Notification) requests
+# empty => Regardless of its origin, a DSN request is always fulfilled.
+#   1   => It can only be requested by authenticated users.
+#   2   => DSN is completely disabled. (not recommended)
+#
+# Note: If this option is left empty and you are using DMARC (reject, quarantine),
+# add an SPF record for the FQDN, since the email won't be signd with DKIM and will
+# be sent from mail.example.com instead of example.com.
+#
+POSTFIX_DSN=
```

Comment:
* If other maintainers decide to keep the ENV, I will gladly to document it.

* This also makes sense and certainly looks nicer.

* As for the note, my idea was to inform users who don't know much about Postfix.
I figured you would just ignore it if you had already changed it to the bare domain, since you had to do it manually.